appveyor.yml

```diff
@@ -0,0 +1,63 @@
+build: false
+platform:
+  - x64
+clone_folder: c:\projects\workspace
+
+environment:
+  matrix:
+  #- dependencies: lowest
```

Comment:
I would add a comment linked to an issue or something here. Otherwise it's just a commented code/config we don't know if it can be removed or will be enabled later or something